openmetadata-ui/src/main/resources/ui/src/components/common/toggle-switch/ToggleSwitchV1.tsx

```diff
@@ -11,29 +11,29 @@
  *  limitations under the License.
  */
 
-import classNames from 'classnames';
+import { Switch } from 'antd';
 import React from 'react';
 
 export interface ToggleSwitchV1Props {
   checked: boolean;
   handleCheck: () => void;
   testId?: string;
+  id?: string;
 }
 
 const ToggleSwitchV1 = ({
   checked,
   handleCheck,
   testId,
+  id,
 }: ToggleSwitchV1Props) => {
-  const id = testId ? `toggle-button-${testId}` : 'toggle-button';
-
   return (
-    <div
-      className={classNames('toggle-switch', checked ? 'open' : null)}
-      data-testid={id}
-      onClick={handleCheck}>
-      <div className="switch" />
-    </div>
+    <Switch
+      checked={checked}
+      data-testid={testId ? `toggle-button-${testId}` : 'toggle-button'}
+      id={id}
+      onChange={handleCheck}
+    />
```

Comment:
We can get rid of this component and directly use Switch since there's no login involved in this component

Reply:
Yes, will remove it.